frontend/src/components/Calendar/Day.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import styles from "./Day.module.css";
+import dayjs from "dayjs";
+
+function Day({ day, row }) {
+  const isToday = day.format("DD-MM-YY") === dayjs().format("DD-MM-YY"); // Check if the day is today
+  const circleColor = isToday ? "#aacaef" : ""; // If it is today, the circle will be blue
+
+  const handleClick = () => {

```

Comment:
- Maybe we can change the color of the date (with lighter color compare to the circleColor) when it is hover over or clicked. Same go for mini day below. We can use google calednar as a reference for this

Reply:
Great idea! I think we are gonna handle clicked buttons later after asking for anh Khoa's guidance on state control but I just added hovering effects